File: res/template.go
package main

import (
	"fmt"
	"io/ioutil"
)

// returns the puzzle input
func get_input() string {
	data, err := ioutil.ReadFile("{{ROOT}}/input.txt")
	if err != nil {
		fmt.Println(err)
	}
	return string(data)
}

// write the solution to the output file
func output(str string) error {
	return ioutil.WriteFile("{{ROOT}}/output.txt", []byte(str), 0777)
}

func main() {
	// Solution goes here
}

File: res/template.rb
# returns the puzzle input
def get_input
  File.open("{{ROOT}}/input.txt").read.chomp
end

# write the solution to the output file
def output(str)
  File.open("{{ROOT}}/output.txt", "w") do |f|
    f.write str
  end
end

# Solution goes here